.github/scripts/backport.py

```diff
@@ -0,0 +1,408 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Backport CLI (label + remind)
+
+Label merged PRs lacking a version label with 'backport pending' or send reminder
+comments on such PRs until a version label (e.g. v9.2) is added.
+
+Quick usage:
+	backport.py help
+	backport.py --event-name pull_request_target label
+	backport.py --repo owner/repo --token TOKEN --event-name schedule label --lookback-days 7
+	backport.py --repo owner/repo --token TOKEN --event-name workflow_dispatch remind --lookback-days 30 --pending-label-age-days 14
```

Comment:
Reading a token should be done from env variable instead of command line. Command line leaves a trace in shell history which can be avoided but requires extra care.